cmd/geth/main.go

```diff
@@ -165,6 +165,7 @@ var (
 		utils.MinerNotifyFullFlag,
 		configFileFlag,
 		utils.CatalystFlag,
+		utils.BlockAmountReserved,
```

Comment:
unnecessry?

Reply:
I think it should be necessary? Because it will initialize the config, which will be used for GlobalFinding  the value of ctx